README.md

```diff
@@ -245,6 +245,31 @@ accessed at [http://localhost:4200](http://localhost:4200):
     # OR
     (docker)# dash npm-start
 
+### Setup system to use the cephadm Vagrant box
+
+Ceph provides a Vagrant box that can be used with this Docker container.
+
+To be able to interact with this Vagrant box you need to start it and
+extract the SSH configuration. Note, your Docker container must be started
+with the `-v ~/.ssh:/root/.ssh:ro` command line arguments to be able to
```

Comment:
@votdev Can we add this to the `setup.sh` script, so `setup-cephadm.sh` will work by default? -  https://github.com/ricardoasmarques/ceph-dev-docker/blob/master/setup.sh#L38